README.md

```diff
@@ -1,5 +1,7 @@
 # [opensourcedesign.net](https://opensourcedesign.net)
 
+(https://github.com/opensourcedesign/opensourcedesign.github.io/stargazers) [![GitHub license](https://img.shields.io/github/license/opensourcedesign/opensourcedesign.github.io?logo=apache)](https://github.com/opensourcedesign/opensourcedesign.github.io/blob/master/LICENSE) ![Twitter Follow](https://img.shields.io/twitter/follow/opensrcdesign?style=social)
+
 Website of the Open Source Design community, hosted on Github pages.
 
 [![Backers on Open Collective](https://opencollective.com/opensourcedesign/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/opensourcedesign/sponsors/badge.svg)](#sponsors)
```

Comment:
The Twitter follow badge should go after the Open Collective badges here. :) That way the description above is also not obscured by badges.

And again – the stargazers and license info we do not need as a badge, as it is shown directly on Github in the already very busy top area ;)
![image](https://user-images.githubusercontent.com/925062/70027611-1a332c00-15d5-11ea-8dbf-d6bc57eaf32f.png)

@vinitshahdeo could you make the changes and also squash your commits into 1? It’s 5 commits by now for just adding a Twitter button, we should keep history clean if possible. :)